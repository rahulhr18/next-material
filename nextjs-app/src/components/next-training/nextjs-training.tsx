import React from 'react'
import { useStyles } from './style'

export const NextJsTraining: React.FC = () => {
  const classes = useStyles()
  return (
    <div
      id="about-training"
      className={classes.wrapper}
    >
      <div className={classes.container}>
        <div className={classes.caresection}>
            <h1>My Awesome shopping site</h1>
            <h2>Men's t-shirt, Sweatshirt</h2>
        </div>
      </div>
    </div>
  )
}
