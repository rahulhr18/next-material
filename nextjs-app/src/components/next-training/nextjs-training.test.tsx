import React from 'react'
import { shallow } from 'enzyme'
import { NextJsTraining } from './nextjs-training'

describe('Next JS training', () => {
  it('matches snapshot', () => {
    const wrapper = shallow(
        <NextJsTraining />
    )
    expect(wrapper).toMatchSnapshot()
  })
})
