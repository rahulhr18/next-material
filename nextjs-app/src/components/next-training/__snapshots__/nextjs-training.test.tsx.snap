// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Next JS training matches snapshot 1`] = `
<div
  className="makeStyles-wrapper-1"
  id="about-training"
>
  <div
    className="makeStyles-container-2"
  >
    <div
      className="makeStyles-caresection-3"
    >
      <h1>
        Next.js Training
      </h1>
      <h4>
        Training Module
      </h4>
    </div>
  </div>
</div>
`;
